### PR TITLE
Remove unused locals

### PR DESCRIPTION
**Pull request by drewnoakes**

### CODE REVIEW

File: src/Microsoft.VisualStudio.Editors/MyApplication/MyApplicationProperties.vb [hunk 1]
```diff
@@ -763,7 +763,7 @@ Namespace Microsoft.VisualStudio.Editors.MyApplication
         ''' <remarks>Caller is responsible for catching exceptions</remarks>
         Private Shared Sub SetCustomTool(ProjectItem As ProjectItem, Value As String)
             Dim ToolProperty As [Property] = GetProjectItemProperty(ProjectItem, PROJECTPROPERTY_CUSTOMTOOL)
-            Dim NamespaceProperty As [Property] = GetProjectItemProperty(ProjectItem, PROJECTPROPERTY_CUSTOMTOOLNAMESPACE)
+            'Dim NamespaceProperty As [Property] = GetProjectItemProperty(ProjectItem, PROJECTPROPERTY_CUSTOMTOOLNAMESPACE)
```
Comment (davkean) [inline src/Microsoft.VisualStudio.Editors/MyApplication/MyApplicationProperties.vb]: Why?

File: tests/Microsoft.VisualStudio.ProjectSystem.Managed.UnitTests/ProjectSystem/LanguageServices/WorkspaceProjectContextProviderTests.cs [hunk 2]
```diff
@@ -169,7 +161,7 @@ public async Task CreateProjectContextAsync_ReturnsContextWithLastDesignTimeBuil
 
             var result = await provider.CreateProjectContextAsync(project);
 
-            Assert.NotNull(context);
+            Assert.NotNull(result);
```
Comment (drewnoakes) [inline tests/Microsoft.VisualStudio.ProjectSystem.Managed.UnitTests/ProjectSystem/LanguageServices/WorkspaceProjectContextProviderTests.cs]: I _think_ this was the intention.

File: src/Microsoft.VisualStudio.Editors/MyApplication/MyApplicationProperties.vb (hunk 1 above)
Comment (drewnoakes) [reply src/Microsoft.VisualStudio.Editors/MyApplication/MyApplicationProperties.vb]: I wondered whether this was highlighting a bug or not. If you think it's fine, I'll remove it.

File: tests/Microsoft.VisualStudio.ProjectSystem.Managed.UnitTests/ProjectSystem/LanguageServices/WorkspaceProjectContextProviderTests.cs (hunk 2 above)
Comment (davkean) [reply tests/Microsoft.VisualStudio.ProjectSystem.Managed.UnitTests/ProjectSystem/LanguageServices/WorkspaceProjectContextProviderTests.cs]: Yes agreed.

File: src/Microsoft.VisualStudio.Editors/MyApplication/MyApplicationProperties.vb (hunk 1 above)
Comment (davkean) [reply src/Microsoft.VisualStudio.Editors/MyApplication/MyApplicationProperties.vb]: I looked at the code, I don't think this an indicator of a bug. The namespace shouldn't be changed.

**Pull request merged**